datalake/src/test/java/com/sequenceiq/datalake/service/sdx/SdxServiceTest.java

```diff
@@ -124,6 +125,9 @@
     @Mock
     private CDPConfigService cdpConfigService;
 
+    @Mock
+    private FlowCancelService flowCancelServicel;
```

Comment:
typo in `flowCancelServicel` -> `flowCancelService`

Reply:
fixed